instance/tests/models/test_openedx_instance.py

```diff
@@ -191,6 +227,26 @@ def test_spawn_appserver(self, mocks, mock_provision):
         self.assertEqual(configuration_vars['EDXAPP_CONTACT_EMAIL'], instance.email)
 
     @patch_services
+    @patch('instance.models.openedx_instance.OpenEdXAppServer.provision', return_value=True)
+    def test_spawn_appserver_with_external_domains(self, mocks, mock_provision):
+        """
+        Test that external domains are used when provisioning a new server.
+        """
+        instance = OpenEdXInstanceFactory(
+            sub_domain='test.spawn', use_ephemeral_databases=True,
+            external_lms_domain='lms.external.com', external_lms_preview_domain='lmspreview.external.com',
+            external_studio_domain='cms.external.com'
+        )
```

Comment:
@mtyaka Nit: This would be even more readable (and easier to edit) if you put each arg on a separate line.